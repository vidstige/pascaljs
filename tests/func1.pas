program func1;
function simple(A, B: Boolean): Boolean;
begin
  simple := A or B;
end;
function NoParameters: Boolean;
begin
  NoParameters := false;
end;
var
  r: Boolean;
begin
  r := simple(true, false);
  WriteLn('A or B ', r);
  WriteLn('NoParameters() ', NoParameters());
end.