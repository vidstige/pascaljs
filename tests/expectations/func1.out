A or B true
NoParameters() false
NoParameters, no parenthesis false
